.github/workflows/default-skipped.yml

```diff
@@ -0,0 +1,20 @@
+name: Skipped Require default
```

Comment:
Let's use consistent naming style for the action workflows.
Please rename it to `default-skipped` for brevity in the PR check list.